src/index.html

```diff
@@ -5,8 +5,322 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__link">
+            <a
+              href="#"
+              class="nav__item is-active">Apple
+            </a>
+          </li>
+
+          <li class="nav__link">
+            <a href="#" class="nav__item">Samsung</a>
+          </li>
+          <li class="nav__link">
+            <a href="#" class="nav__item">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__link">
+            <a
+              href="#"
+              class="nav__item"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__link">
+            <a href="#" class="nav__item">Гаджеты</a>
+          </li>
+
+          <li class="nav__link">
+            <a href="#" class="nav__item">Планшеты</a>
+          </li>
+
+          <li class="nav__link">
+            <a href="#" class="nav__item">Фото</a>
+          </li>
+
+          <li class="nav__link">
+            <a href="#" class="nav__item">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div class="card-display">
+        <section class="card" data-qa="card">
```

Comment:
Delete unnecessary wrapper `card-display`. You could use for this purposes mix.
More details in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)

And add changes to styles according to changes in html.

```suggestion
    <main class="catalog">
        <section class="card catalog__card" data-qa="card">
```